lms/djangoapps/certificates/signals.py

```diff
@@ -52,10 +52,10 @@ def _update_cert_settings_on_pacing_change(sender, updated_course_overview, **kw
     ))
 
 
-@receiver(post_save, sender=CertificateWhitelist, dispatch_uid="append_certificate_whitelist")
-def _listen_for_certificate_whitelist_append(sender, instance, **kwargs):  # pylint: disable=unused-argument
+@receiver(post_save, sender=CertificateWhitelist, dispatch_uid="append_certificate_allowlist")
+def _listen_for_certificate_allowlist_append(sender, instance, **kwargs):  # pylint: disable=unused-argument
```

Comment:
I did not create a temporary duplicate function for this here as it doens't look like it's imported/called anywhere. The mechanism of being invoked as a post_save trigger/signal on the CertificateWhitelist model, so I don't think we're in any danger here from a blue/green deployment standpoint.